sentences/fr/_common.yaml

```diff
@@ -75,19 +75,22 @@ lists:
   cover_classes:
     values:
       - in: "store[s]"
-        out: "awning"
-      - in: "store[s]"
+        out:
+          - "awning"
+          - "blind"
+          - "shade"
+      - in: "store[s] vénitien[s]"
         out: "blind"
+      - in: "(banne[s])|(auvent[s])|(store[s] banne[s])"
+        out: "awning"
       - in: "rideau[x]"
         out: "curtain"
       - in: "porte[s]"
         out: "door"
       - in: "portes[s] de garage[s]"
-        out: "garage door"
```

Comment:
@piitaya In case yourself ask why I changed that one seemingly randomly: 
![image](https://github.com/home-assistant/intents/assets/5878296/11213bdf-46e2-414d-8871-b014e6446381)

The correct device class of a garage door is indeed `garage`